lib/backend/repository/repository.go

```diff
@@ -302,7 +302,7 @@ func getLayer(imgID, registry string, repoData *RepoData, imgSize int64, tmpDir
 	progressReader := &ioprogress.Reader{
 		Reader:       res.Body,
 		Size:         imgSize,
-		DrawFunc:     ioprogress.DrawTerminalf(os.Stdout, fmtfunc),
+		DrawFunc:     ioprogress.DrawTerminalf(os.Stderr, fmtfunc),
```

Comment:
This is a lib and really shouldn't be printing to arbitrary streams - it should probably accept an optional io.Writer that users of the lib can inject. Would you mind filing a followup issue?